frontend/src/component/playground/Playground/AdvancedPlayground.test.tsx

```diff
@@ -22,6 +24,18 @@ const testComponent = (
     </UIProviderContainer>
 );
 
+const testEvaluateComponent = (
+    <UIProviderContainer>
+        <AdvancedPlayground
+            FormComponent={props => (
+                <form onSubmit={props.onSubmit}>
```

Comment:
instead of using real form with hard to test code mirror I use the test form. The important stuff is that it calls onSubmit from the parent component owning the state.